plugins/woocommerce/client/admin/client/settings-payments/onboarding/providers/woopayments/steps/test-or-live-account/style.scss

```diff
@@ -0,0 +1,74 @@
+.woocommerce-payments-test-or-live-account-step {
+	display: flex;
+	flex-direction: column;
+	height: 100%;
+
+	.center-content {
+		text-align: center;
+	}
+
+	&__success_content_title {
```

Comment:
We can use `&_title` and nest the selector under the `&__success_content` block.